extensions/ql-vscode/package.json

```
@@ -595,19 +595,19 @@
       },
       {
         "command": "codeQLQueryResults.down",
-        "title": "CodeQL: Navigate down in result viewer"
+        "title": "CodeQL: Navigate Down in Result Viewer"
       },
       {
         "command": "codeQLQueryResults.up",
-        "title": "CodeQL: Navigate up in result viewer"
+        "title": "CodeQL: Navigate Up in Result Viewer"
       },
       {
         "command": "codeQLQueryResults.right",
-        "title": "CodeQL: Navigate right in result viewer"
+        "title": "CodeQL: Navigate Right in Result Viewer"
       },
       {
         "command": "codeQLQueryResults.left",
-        "title": "CodeQL: Navigate left in result viewer"
+        "title": "CodeQL: Navigate Left in Result Viewer"
```

We also have the MRVA result  viewer, which this change doesn't support.

```suggestion
        "title": "CodeQL: Navigate Down in Local Result Viewer"
      },
      {
        "command": "codeQLQueryResults.up",
        "title": "CodeQL: Navigate Up in Local Result Viewer"
      },
      {
        "command": "codeQLQueryResults.right",
        "title": "CodeQL: Navigate Right in Local Result Viewer"
      },
      {
        "command": "codeQLQueryResults.left",
        "title": "CodeQL: Navigate Left in Local Result Viewer"
```